tests/test_layer.py

```diff
@@ -869,5 +871,158 @@ def helper_test_add_modeling_region(layer):
             pytest.fail(str(e))
 
 
+def helper_test_update_modeling_region(layer):
+    modeling_region_example = [
+        {
+            "cca_name": "Card",
+            "region_id": "Region001",
```

Comment:
It isn't obvious that this region ID must be the same as the one used to create the modeling region in a previous test.
One way to improve this is to have `helper_test_add_modeling_region()` return the region ID and add a parameter to `helper_test_update_modeling_region()`.
This is similar to other cases like this. See `test_lifecycle.test_all()` for examples.

Doing it this way better ensures that the test methods are called in the right order and makes the dependencies between tests clearer.